KLSadd_insertion/src/updateChapters.py

```diff
@@ -1,110 +1,327 @@
+    chap.insert(foot_misc_index, "\\usepackage[pdftex]{hyperref} \n\\usepackage {xparse} \n\\usepackage{cite} \n")
     return chap
 
-# 2/18/16 this method reads in relevant commands that are in KLSadd.tex
-# and returns them as a list
 
+def get_commands(kls, new_commands):
+    """
+    This method reads in relevant commands that are in KLS Addendum.tex and returns them as a list
 
-def getCommands(kls):
+    :param kls: The addendum is the source of what is being found
+    :return:
+    """
     index = 0
     for word in kls:
         index += 1
-        if("smallskipamount" in word):
-            newCommands.append(index - 1)
-        if("mybibitem[1]" in word):
-            newCommands.append(index)
-    comms = kls[newCommands[0]:newCommands[1]]
-    return comms
+        if "smallskipamount" in word:
+            new_commands.append(index - 1)
+        if "mybibitem[1]" in word:
+            new_commands.append(index)
+    print new_commands
+    return kls[new_commands[0]:new_commands[1]]
 
-# 2/18/16 this method addresses the goal of hardcoding in the necessary
-# commands to let the chapter files run as pdf's. Currently only works
-# with chapter 9
 
+# 2/18/16 this method addresses the goal of hardcoding in the necessary commands to let the chapter files run as pdf's.
+# Currently only works with chapter 9
+def insert_commands(kls, chap, cms):
+    """
+    Inserts commands identified in previous functions.
 
-def insertCommands(kls, chap, cms):
+    :param kls: Addendum to look through
+    :param chap: The chapter that is receiving commands (9 or 14)
+    :param cms: Commands to be inserted
+    :return:
+    """
     # reads in the newCommands[] and puts them in chap
-    # the index of the "begin document" keyphrase, this is where the new
-    # commands need to be inserted.
-    beginIndex = -1
-    # find index of begin document in KLSadd
+    begin_index = -1  # the index of the "begin document" keyphrase, this is where the new commands need to be inserted.
+    # find index of begin document in KLS Addendum
     index = 0
+
     for word in kls:
         index += 1
-        if("begin{document}" in word):
-            beginIndex += index
+        if "begin{document}" in word:
+            begin_index += index
     tempIndex = 0
+
     for i in cms:
-        chap.insert(beginIndex + tempIndex, i)
+        chap.insert(begin_index + tempIndex, i)
         tempIndex += 1
     return chap
 
-# method to find the indices of the reference paragraphs
 
+# method to find the indices of the reference paragraphs
+def find_references(chapter, chapticker, math_people):
+    """
+    This function searches the chapters and locates potential destinations of additions from the addendum.
+    It puts these destinations in a list.
 
-def findReferences(chapter):
+    :param chapter: The chapter being searched.
+    :param chapticker: Which chapter is being searched (9 or 14).
```

Comment:
Considering this, why is its value always either 0 or 1? Why not just use 9 or 14?